components/Chips/examples/ChipsInputExampleViewController.m

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ChipsExamplesSupplemental.h"
+
+#import "MaterialChips.h"
+#import "MaterialTextFields.h"
+
+@interface ChipsInputExampleViewController () <MDCChipFieldDelegate>
+@end
+
+@implementation ChipsInputExampleViewController {
+  MDCChipField *_chipField;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.view.backgroundColor = [UIColor lightGrayColor];
+
+  _chipField = [[MDCChipField alloc] initWithFrame:CGRectZero];
```

Comment:
Relying on the default delimiter doesn't seem to work in the simulator in this example. I had to set the delimiter to a non-default to get this to work (using iOS 11.2 in the simulator).

Reply:
Ah it should now. I had disabled the delimiter in the example for testing.